Apps/phawscloudtrail/awscloudtrail.json

```diff
@@ -11,7 +11,7 @@
     "python_version": "3",
     "publisher": "Splunk",
     "license": "Copyright (c) 2019-2021 Splunk Inc.",
-    "app_version": "2.0.0",
+    "app_version": "2.1.0",
```

Comment:
What is the logic for incrementing the version?

Reply:
We need to bump either major or minor for GA releases and I wasn't sure this merited a major version bump since it's just dependencies being updated.